sites/public/src/pages/applications/household/member.tsx

```diff
@@ -386,9 +429,13 @@ const ApplicationMember = () => {
                       label={t("application.contact.zip")}
                       placeholder={t("application.contact.zipCode")}
                       defaultValue={member.workAddress.zipCode}
-                      validation={{ required: true }}
+                      validation={{ required: true, maxLength: 64 }}
```

Comment:
I'm wondering if ones that should never be close to 64 characters we should reduce the number? It would require a different error message. For now though since this is the number the backend is capped at we can keep them all at 64

Reply:
Sounds good! Makes sense to revisit if 64 makes sense in the backend.